.github/workflows/main.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     - name: Setup Node.js for use with actions
       uses: actions/setup-node@v1
       with:
-        node-version: 16.x
+        node-version: 14.x
```

Comment:
Is the node downgrade required while at the same time bumping the ES target forward from 2015 to 2020? It looks like node 14 is the minimum version for supporting ES2020 but do we need to revert at all?

Reply:
cool. reverted.  still have a mistmatch between dockerfile and ci but can address that somewhere else.  (also are tests are limited so probably doesn't matter)